tests/components/raycaster.test.js

```diff
@@ -120,12 +156,105 @@ suite('raycaster', function () {
       });
 
       targetEl.setAttribute('geometry', 'primitive: box; depth: 1; height: 1; width: 1;');
-      targetEl.setAttribute('material', '');
       targetEl.setAttribute('position', '0 0 -1');
-      targetEl.addEventListener('loaded', function finishSetup () {
+      el.sceneEl.appendChild(targetEl);
+      // `npm run test:forefox` needs the timeout for the tests to succeed.
```

Comment:
Probably not worth mentioning, appendChild is asynchronous so it's a good idea to have always.